builtin/game/fine_pointed.lua

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
It is just a local function. Is a verbose name really necessary? I don't oppose that, but I prefer shorter (but not to short) function names and add a short comment. To long function names make code confusing